app/models/maths_and_physics/eligibility_admin_answers_presenter.rb

```diff
@@ -0,0 +1,45 @@
+module MathsAndPhysics
+  class EligibilityAdminAnswersPresenter
+    include Admin::PresenterMethods
+
+    attr_reader :eligibility
+
+    def initialize(eligibility)
+      @eligibility = eligibility
+    end
+
+    # Formats the eligibility as a list of questions and answers.
+    # Suitable for playback to the service operators for them to review
+    # the claim.
+    #
+    # Returns an array. Each element of this an array is an array of two
+    # elements:
+    # [0]: short question text;
+    # [1]: answer text;
+    def answers
+      [].tap do |a|
+        a << [I18n.t("maths_and_physics.admin.teaching_maths_or_physics"), (eligibility.teaching_maths_or_physics? ? "Yes" : "No")]
+        a << [I18n.t("admin.current_school"), display_school(eligibility.current_school)]
+        a << [I18n.t("maths_and_physics.admin.initial_teacher_training_subject"), I18n.t("maths_and_physics.answers.initial_teacher_training_subject.#{eligibility.initial_teacher_training_subject}")]
+        a << [I18n.t("maths_and_physics.admin.initial_teacher_training_subject_specialism"), I18n.t("maths_and_physics.answers.initial_teacher_training_subject_specialism.#{eligibility.initial_teacher_training_subject_specialism}")] if eligibility.initial_teacher_training_subject_specialism.present?
+        a << [I18n.t("maths_and_physics.admin.has_uk_maths_or_physics_degree"), degree_answer] if eligibility.has_uk_maths_or_physics_degree.present?
+        a << [I18n.t("admin.qts_award_year"), I18n.t("maths_and_physics.questions.qts_award_years.#{eligibility.qts_award_year}")]
+        a << [I18n.t("maths_and_physics.admin.employed_as_supply_teacher"), (eligibility.employed_as_supply_teacher? ? "Yes" : "No")]
+        a << [I18n.t("maths_and_physics.admin.has_entire_term_contract"), (eligibility.has_entire_term_contract? ? "Yes" : "No")] if eligibility.has_entire_term_contract.present?
```

Comment:
I think that for a boolean, `#nil?` communicates intent more clearly than `#present?`, which returns true when called on `false`. In this particular example, it probably doesn't make a difference, because you can't submit a claim if this value is false. But I think it would set a better example for future, similar, questions.

Reply:
Good point, I've updated the code to use `unless nil?`